library/src/main/resources/pages/table-footer.html

```diff
@@ -0,0 +1,8 @@
+[#ftl]
```

Comment:
@gvjoshi25 where is table header, pager, and footer used?

Reply:
They are not in use. I removed them.